tests/testSpherePoint.py

```diff
@@ -773,9 +778,9 @@ def testReprValue(self):
             self.assertIn("degrees", pointRepr)
             self.assertEqual(2, len(pointRepr.split(",")))
 
-            copy = eval(pointRepr)
-            self.assertAnglesNearlyEqual(point.getLongitude(), copy.getLongitude())
-            self.assertAnglesNearlyEqual(point.getLatitude(), copy.getLatitude())
+            spcopy = eval(pointRepr)
+            self.assertAnglesNearlyEqual(point.getLongitude(), spcopy.getLongitude())
+            self.assertAnglesNearlyEqual(point.getLatitude(), spcopy.getLatitude())
```

Comment:
Yes. `copy` hides a global function name so I picked one of the other names that was already being used for a copy of a point (in a different test in this file).